b2sdk/bucket.py

```diff
@@ -115,17 +115,16 @@ def set_type(self, bucket_type):
 
     def update(
         self,
-        bucket_type=None,
-        bucket_info=None,
-        cors_rules=None,
-        lifecycle_rules=None,
-        if_revision_is=None,
+        bucket_type: Optional[str] = None,
+        bucket_info: Optional[dict] = None,
+        cors_rules: Optional[dict] = None,
+        lifecycle_rules: Optional[dict] = None,
+        if_revision_is: Optional[int] = None,
         default_server_side_encryption: Optional[EncryptionSetting] = None,
         default_retention: Optional[BucketRetentionSetting] = None,
     ):
         """
         Update various bucket parameters.
-        For legacy reasons in apiver v1 it returns whatever server returned on b2_update_bucket call, v2 will change that.
 
         :param str bucket_type: a bucket type
```

Comment:
maybe worth mentioning allPublic and allPrivate here, since it's not easily available anywhere near here